src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj

```diff
@@ -1,13 +1,10 @@
 ﻿
 <Project Sdk="Microsoft.NET.Sdk">
-    <!-- Please do not move/edit code below this line -->
-    <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-    <!-- Please do not move/edit code below this line -->
-
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <PackageId>Microsoft.AzureStack.Management.Fabric.Admin</PackageId>
     <Description>Provides developers with libraries for the fabric admin under AzureStack to manage fabric infrastructure.</Description>
-    <VersionPrefix>0.1.1-preview</VersionPrefix>
+    <VersionPrefix>0.2.0-preview</VersionPrefix>
     <AssemblyName>Microsoft.AzureStack.Management.Fabric.Admin</AssemblyName>
```

Comment:
Please add [these](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L23) lines at the bottom of the csproj

Reply:
Done.